src/smt/smt_engine.cpp

```diff
@@ -4215,9 +4202,13 @@ void SmtEnginePrivate::processAssertions() {
     Trace("smt-proc") << "SmtEnginePrivate::processAssertions() : pre-static-learning" << endl;
     // Perform static learning
     Chat() << "doing static learning..." << endl;
-    Trace("simplify") << "SmtEnginePrivate::simplify(): "
+    Trace("simplify") << "SmtEnginePrivate::processAssertions(): "
                       << "performing static learning" << endl;
-    staticLearning();
+    d_smt.finalOptionsAreSet();
```

Comment:
We assert in the beginning of `processAssertions()` that `Assert(d_smt.d_fullyInited);`, so `finalOptionsAreSet()` will not do anything and can be removed.